### PR TITLE
Link to the 7.14.0 ES Python client docs.

### PR DESCRIPTION
**Pull request by michaelbaamonde**
This is the version that Rally depends on, so pointing users elsewhere could be misleading.

### CODE REVIEW

**Comment by michaelbaamonde** [comment]
@elasticmachine test this

**Comment by michaelbaamonde** [comment]
@elasticmachine run tests please

**Comment by michaelbaamonde** [comment]
Test failures appear to be due to a network error when attempting to build the Docker image.

**Comment by michaelbaamonde** [comment]
@elasticmachine run tests please

**Comment by michaelbaamonde** [comment]
@elasticmachine run tests please

File: docs/command_line_reference.rst [hunk 1]
```diff
@@ -663,7 +663,7 @@ By default, the command line reporter will print the results only on standard ou
 
 With this option you can customize Rally's internal Elasticsearch client.
 
-It accepts a list of comma-separated key-value pairs. The key-value pairs have to be delimited by a colon. These options are passed directly to the Elasticsearch Python client API. See `their documentation on a list of supported options <http://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch>`_.
+It accepts a list of comma-separated key-value pairs. The key-value pairs have to be delimited by a colon. These options are passed directly to the Elasticsearch Python client API. See `their documentation on a list of supported options <https://elasticsearch-py.readthedocs.io/en/v7.14.0/api.html#elasticsearch>`_.
```
Comment (dliappis) [inline docs/command_line_reference.rst:666]: For the sake of not having to remember changing the version here when we upgrade `elasticsearch-py` could we leverage a variable, similar to how we do for [MIN_PY_VER](https://github.com/elastic/rally/blame/master/docs/install.rst#L30)? This is achieved with real python code on [docs/conf.py](https://github.com/elastic/rally/blob/a921dccd3158f3821d36d74bcc9e19b629594f1e/docs/conf.py#L49-L59).

**Review by dliappis — COMMENTED** [review]
Good catch. Left a comment that could help reduce maintenance burden in the future.

File: docs/command_line_reference.rst (hunk 1 above)
Comment (michaelbaamonde) [reply docs/command_line_reference.rst:666]: Perfect, didn't even occur to me. I'll do that.

Comment (michaelbaamonde) [reply docs/command_line_reference.rst]: Done in https://github.com/elastic/rally/pull/1554/commits/8fb9a3dbe2e9e1b4cf369deb43a857307495e46a

**Comment by michaelbaamonde** [comment]
@elasticmachine run tests please

**Review by DJRickyB — APPROVED** [review]
LGTM. You may need to forcibly merge given the checks have changed while this has been open; no need of further review from me on that

**Review by inqueue — APPROVED** [review]
LGTM

**Pull request merged**